modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -253,6 +260,35 @@ namespace imgproc {
 
 }
 
+// TODO: move to the separate file modules/gapi/include/opencv2/gapi/video.hpp
+namespace  video {
+    using GPoint2fArray        = cv::GArray<cv::Point2f>;
+    using GUcharArray          = cv::GArray<uchar>;
+    using GFloatArray          = cv::GArray<float>;
+    using GGMatArray           = cv::GArray<cv::GMat>;
+
+    using GOptFlowLKOutput     = std::tuple<GPoint2fArray, GUcharArray, GFloatArray>;
+    using GOptFlowLKOutputDesc = std::tuple<GArrayDesc, GArrayDesc, GArrayDesc>;
+
+    G_TYPED_KERNEL(GCalcOptFlowLK, <GOptFlowLKOutput(GMat,GMat,GPoint2fArray,GPoint2fArray,
+                                                     Size,int,TermCriteria,int,double)>,
+                   "org.opencv.video.calcOpticalFlowLK") {
```

Comment:
let's not confuse people, it always works with Pyramids and the OpenCV kernel is always `-PyrLK`.

Reply:
ok